float64 error
float64 error_proportional
float64 error_integral
float64 integral_sum
float64 error_derivative
float64 derivative
int64 episodes
int64 time_steps

